cmd/cli/serve/full.go

```diff
@@ -153,16 +153,6 @@ func fullServer(cmd *cobra.Command, _ []string) error {
 		return fmt.Errorf("parsing config: %w", err)
 	}
 
-	// Set graceful shutdown timeout.
-	// This timeout allows fx to wait up to one minute for all lifecycle shutdown hooks to execute.
-	// Here is how we arrived at this value:
-	//  - Proving a piece typically takes 30 seconds, so a min is plenty there
-	//  - Assuming a symmetric 100Mbps connection, upload/download of 256 MB (max piece size) will likely take 20-30 seconds
-	//    therefore, 1 min should be enough time to let existing connections complete
-	// Still, if any of the above take more than 1min, they will be closed/rejected after 1min, so we might want
-	// to make this a configuration value based on user preference, metrics we collect, and capacity of the machine.
-	shutdownTimeout := time.Minute
```

Comment:
This got moved into the cli/setup dir as the systemctl file needs to be aware of this value when timing out a shutdown.